### DISCUSSION TITLE
Per-anchor stats grouped by page

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1699] [Anything Else]
"Is there any way of getting per-anchor analytics on a page? e.g. a breakdown by blahblah.html#first blahblah.html#second. I really would love the ability to see results grouped by page, and then be able to go into those and see which anchors were being visited within that page. Without the grouping it makes it harder to see trends right now"

([via twitter](https://twitter.com/_benui/status/1495793002350268416))

**Comment by benui-dev** [comment]
For more context, per-anchor stats are supported this way:
https://plausible.io/docs/hash-based-routing

With this enabled, all page visits are separate by anchor.
